src/electron/window.main.ts

```diff
@@ -81,7 +81,13 @@ export class WindowMain {
         });
     }
 
-    private async createWindow() {
+    async recreateWindow(): Promise<void> {
+        if (this.win === null) {
+            await this.createWindow();
+        }
+    }
+
+    private async createWindow(): Promise<void> {
```

Comment:
Couldn't you just make this public so that you can call it?

Reply:
Will do, thanks for the feedback. :)